EIPS/eip-7401.md

```diff
@@ -4,13 +4,19 @@ title: Parent-Governed Non-Fungible Tokens Nesting
 description: An interface for Non-Fungible Tokens Nesting with emphasis on parent token's control over the relationship.
 author: Bruno Škvorc (@Swader), Cicada (@CicadaNCR), Steven Pineda (@steven2308), Stevan Bogosavljevic (@stevyhacker), Jan Turk (@ThunderDeliverer)
 discussions-to: https://ethereum-magicians.org/t/eip-6059-parent-governed-nestable-non-fungible-tokens/11914
-status: Draft
+status: Review
 type: Standards Track
 category: ERC
 created: 2023-07-26
 requires: 165, 721
 ---
 
+> :information_source: **[ERC-7401](./eip-7401.md) has superseded [ERC-6059](./eip-6059.md).** :information_source:  
+> [ERC-7401](./eip-7401.md) fixes the inconsistency in the [ERC-6059](./eip-6059.md) interface specification, where interface ID doesn't match the interface specified as the interface evolved during the proposal's lifecycle, but one of the parameters was not added to it. The missing parameter is, however, present in the interface ID.
+> Apart from this fix, [ERC-7401](./eip-7401.md) is functionally equivalent to [ERC-6059](./eip-6059.md).
+>
+> :warning: [ERC-7401](./eip-7401.md) MUST be used in lieu of [ERC-6059](./eip-6059.md). :warning:
+
```

Comment:
`:information_source:` doesn't actually render to anything. I guess use an emoji?

I'd prefer this to be moved into the motivation section, but I won't be too strict about that.